app/views/file_attachments/index.html.erb

```diff
@@ -39,7 +39,17 @@
       </h2>
 
       <% attachments.each do |attachment| %>
-        <%= render "file_attachment", attachment: attachment, document: @edition.document %>
+        <%= render "components/attachment_meta", {
+          title: attachment.title,
+          url: "#",
+          metadata_items: [
+            attachment.blob.content_type,
+            number_to_human_size(attachment.blob.byte_size),
```

Comment:
We should already have both of these without needing to go through blob.

We already have content_type as a method on a blob_revision which should be delegated to revision (as per [Image::Revision](https://github.com/alphagov/content-publisher/blob/master/app/models/image/revision.rb#L31) and should probably also delegate byte_size in the same way.

We have a size method on a FileAttachment::BlobRevision - which was used for this first. However I imagine Emma and I had overlooked that it was available on a blob so that should be removed if we're just going to use the blob one: https://github.com/alphagov/content-publisher/blob/03d45f97fc9b78e6b71fed1f7e0160c0300c914f/db/schema.rb#L109

Reply:
@kevindew ah, totally missed that. I've done as you suggested, and added a commit to remove the redundant field on the revision.